modules/serverpod_auth/serverpod_auth_server/lib/src/business/config.dart

```diff
@@ -138,6 +145,12 @@ class AuthConfig {
   /// generation.
   final bool allowUnsecureRandom;
 
+  /// Create a custom hash for the password
+  final GeneratePasswordHashCallback? generatePasswordHashCallback;
+
+  /// Create a custom validation for the password in combinaison with [GeneratePasswordHashCallback]
+  final ValidatePasswordHashCallback? validatePasswordHashCallback;
```

Comment:
Suggestion: Make these non-nullable and always assign them to the default generator and validator.

No need to run a null check for these in runtime. We might as well always assign them to the default methods.